client/src/utils/router.constant.js

```diff
@@ -0,0 +1,10 @@
+export const HOME_PAGE = '/';
+export const BLOGS = '/blog';
+export const SIGNIN_PAGE = '/signin';
+export const SIGNUP_PAGE = '/signup';
+export const PROFILE = '/profile';
+export const CONTACTS = '/contact';
+export const BOOK = '/book';
+export const SERVICES = '/services';
+export const DASHBOARD = '/dashboard';
+export const DASHBOARD_APPOINTMENT = 'dashboard/appointment	';
```

Comment:
Good job,
- it's the first time see this pattern, can you explain  how the component will render (please  create a dummy skeleton on each page and show us how to render the page),
- don't forget the `404` page.

Reply:
Thanks, 

- the component will render in ``` <Route> ```  component when the Route path ``` Ex :  <Route path= {Home_page} > ```
match the current URL, it renders its children ( ``` <Home/> ``` component )

- ok I'll create Now